task/04-arrays-tasks.js

```diff
@@ -337,7 +338,7 @@ function sortDigitNamesByNumericOrder(arr) {
  *   [ 1, 10, 100, 1000 ]  => 1111
  */
 function getItemsSum(arr) {
-   throw new Error('Not implemented');
+   return arr.length === 0 ? 0 : arr.reduce((a, b) => a + b);
```

Comment:
ternary operator is redundant here
Please, just use initial value

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce